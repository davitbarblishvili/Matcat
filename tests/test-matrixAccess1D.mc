// Author: Davit
func main() int{
    matrix a; 
    a = [[1,2,3],[4,5,6]];
    printm(a[1,:]);

    return 0;
}