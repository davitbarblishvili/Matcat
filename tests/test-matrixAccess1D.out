(
[4.00 5.00 6.00 ]
)